pkg/container/opts_test.go

```diff
@@ -0,0 +1,64 @@
+package container
+
+import "testing"
+
+func Test_cruToLimit(t *testing.T) {
+	type args struct {
+		cru      uint
+		totalCPU int
+	}
+	tests := []struct {
+		name       string
+		args       args
+		wantQuota  int64
+		wantPeriod uint64
+	}{
+		{
+			name: "1-1",
+			args: args{
+				cru:      1,
+				totalCPU: 1,
+			},
+			wantPeriod: 1000000,
+			wantQuota:  1000000,
+		},
+		{
+			name: "1-4",
+			args: args{
+				cru:      1,
+				totalCPU: 4,
+			},
+			wantPeriod: 250000,
```

Comment:
afaict , Period is the reference and the quota is how much of the reference you want to take
hence, numbers need to be switched

Reply:
so period = 1000000, quota = 250000

Reply:
indeed nice catch